tao_compiler/mlir/disc/transforms/disc_passes.td

```diff
@@ -66,10 +66,10 @@ def DiscSpecializeFusionWithSpeculationPass : Pass<"disc-specialize-fusion-with-
   let summary = "Pass for specializing a fusion op with some speculations.";
   let constructor = "createDiscSpecializeFusionWithSpeculationPass()";
   let options = [
-    Option<"cc_major_", "gpu-sm-cc-major", "int",
-            /*default=*/"7", "gpu sm cc_major.">,
-    Option<"cc_minor_", "gpu-sm-cc-minor", "int",
-            /*default=*/"5", "gpu sm cc_minor.">,
+    Option<"core_count_", "core-count", "int",
```

Comment:
sm_count, max_threads_per_sm ?

Reply:
The TF code uses `core` to represent SM on NVIDIA GPU. This pass also works for other GPUs like ROCm, thus it could be better to use a general name.